jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -1194,9 +1194,13 @@ private State parseHeaderStart(ByteBuffer buffer)
                     case CR ->
                     {
                         // Ignore CR and loop around;
+                        crFlag = true;
                     }
                     case LF ->
                     {
+                        if (!crFlag)
```

Comment:
The `crFlag` will already have been set to false by `next(buffer)`

Do we have a test for this violation?

Reply:
Yes, we have 3 tests.

* MultiPartFormDataTest.testLFOnlyEOL()
* MultiPartFormDataTest.testContentTransferEncodingQuotedPrintable()
* MultiPartFormDataTest.testContentTransferEncodingBase64()